apps/curse.cpp

```diff
@@ -23,8 +23,9 @@
 #include <hexer/HexIter.hpp>
 #include <hexer/Processor.hpp>
 #include <hexer/Utils.hpp>
-#include "las.hpp"
 #include "ProgramArgs.hpp"
+#include "../lazperf/readers.hpp"
```

Comment:
You should set the include paths in the CMake configuration instead of depending on these files being in relatives paths.